arithmetization/src/main/java/net/consensys/linea/zktracer/module/limits/precompiles/EcAddEffectiveCall.java

```diff
@@ -59,13 +59,11 @@ public void popTransaction() {
   public void tracePreOpcode(MessageFrame frame) {
     final OpCode opCode = hub.opCode();
 
-    switch (opCode) {
-      case CALL, STATICCALL, DELEGATECALL, CALLCODE -> {
-        final Address target = Words.toAddress(frame.getStackItem(1));
-        if (target.equals(Address.ALTBN128_ADD)
-            && hub.transients().op().gasAllowanceForCall() >= PRECOMPILE_GAS_FEE) {
-          this.counts.push(this.counts.pop() + 1);
-        }
+    if (opCode.isCall()) {
```

Comment:
In actuality we should also check that CALL_DATA_SIZE is nonzero. Indeed the arithmetization doesn't trigger ECDATA otherwise.

Reply:
The same comment applies for
- ECRECOVER
- ECMUL
- ECPAIRING

Reply:
we have to review all the precompile triggering anyway, and maybe trigger them with the HUB, not at `tracePreOpcode` ... cf https://github.com/Consensys/linea-arithmetization/issues/686